build/Targets/Imports.targets

```diff
@@ -353,7 +353,7 @@
              Condition="!Exists('$(ToolsetCompilerPropsFilePath)')" />
   </Target>
 
-  <Target Name="CheckBootstrapState" Condition="'$(BootstrapBuildPath)' != ''" BeforeTargets="CoreCompile">
+  <Target Name="CheckBootstrapState" Condition="'$(BootstrapBuildPath)' != ''" AfterTargets="CoreCompile">
```

Comment:
I suspect I know this is to ensure we see all the load failures that happen during CoreCompile. A comment to that effect might be good.